2025-04-08 15:45:52,222 INFO    MainThread:8128 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2025-04-08 15:45:52,222 INFO    MainThread:8128 [wandb_setup.py:_flush():79] Configure stats pid to 8128
2025-04-08 15:45:52,222 INFO    MainThread:8128 [wandb_setup.py:_flush():79] Loading settings from C:\Users\Hao\.config\wandb\settings
2025-04-08 15:45:52,222 INFO    MainThread:8128 [wandb_setup.py:_flush():79] Loading settings from C:\Users\Hao\Downloads\LLM\Reasoning\wandb\settings
2025-04-08 15:45:52,222 INFO    MainThread:8128 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2025-04-08 15:45:52,222 INFO    MainThread:8128 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2025-04-08 15:45:52,224 INFO    MainThread:8128 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-04-08 15:45:52,224 INFO    MainThread:8128 [wandb_setup.py:_flush():79] Applying login settings: {}
2025-04-08 15:45:52,224 INFO    MainThread:8128 [wandb_init.py:_log_setup():533] Logging user logs to C:\Users\Hao\Downloads\LLM\Reasoning\wandb\run-20250408_154552-56t2s2fc\logs\debug.log
2025-04-08 15:45:52,224 INFO    MainThread:8128 [wandb_init.py:_log_setup():534] Logging internal logs to C:\Users\Hao\Downloads\LLM\Reasoning\wandb\run-20250408_154552-56t2s2fc\logs\debug-internal.log
2025-04-08 15:45:52,224 INFO    MainThread:8128 [wandb_init.py:init():619] calling init triggers
2025-04-08 15:45:52,225 INFO    MainThread:8128 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {'_fields': {'vocab_size': 16, 'device': 'cuda', 'work_dir': 'results', 'batch_size': 32, 'test_size': 2048, 'task': max_seq_len: 83
max_variables: 12
, 'model': activation: &id001 !!python/tuple
- true
- true
- true
bias: false
dropout: 0.1
emb_dim: 64
ff_dim: 256
flash: true
layer_norm: true
mask: true
mlp: *id001
mlp_bias: true
num_heads: !!python/tuple
- 1
- 1
- 1
num_layers: 3
pos_enc: rotary
pos_max_len: 83
, 'training': eval_iter: 100
freeze_out: false
freeze_value: false
get_attn: 1000
get_checkpoints: 500
identity_query: false
label_smoothing: 0
lr: 0.0005
optimizer: adamw
schedule: triangle
total_steps: 60000
warmup_steps: 20000
weight_decay: 0.01
, 'wandb': project: Reasoning
}, '_locked': False, '_type_safe': True, '_convert_dict': True, '_allow_dotted_keys': False, '_sort_keys': True}
2025-04-08 15:45:52,225 INFO    MainThread:8128 [wandb_init.py:init():644] re-initializing run, found existing run on stack: 3ov3yvrd
2025-04-08 15:45:52,226 INFO    MainThread:8128 [wandb_run.py:_finish():2155] finishing run hyan84-university-of-wisconsin-madison/Reasoning/3ov3yvrd
2025-04-08 15:45:52,227 INFO    MainThread:8128 [jupyter.py:save_history():459] not saving jupyter history
2025-04-08 15:45:52,227 INFO    MainThread:8128 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2025-04-08 15:45:52,227 INFO    MainThread:8128 [wandb_init.py:_jupyter_teardown():461] cleaning up jupyter logic
2025-04-08 15:45:52,227 INFO    MainThread:8128 [wandb_run.py:_atexit_cleanup():2420] got exitcode: 0
2025-04-08 15:45:52,227 INFO    MainThread:8128 [wandb_run.py:_restore():2402] restore
2025-04-08 15:45:52,227 INFO    MainThread:8128 [wandb_run.py:_restore():2408] restore done
2025-04-08 15:45:55,001 INFO    MainThread:8128 [wandb_run.py:_footer_history_summary_info():3960] rendering history
2025-04-08 15:45:55,001 INFO    MainThread:8128 [wandb_run.py:_footer_history_summary_info():3992] rendering summary
2025-04-08 15:45:55,009 INFO    MainThread:8128 [wandb_run.py:_footer_sync_info():3921] logging synced files
2025-04-08 15:45:55,012 INFO    MainThread:8128 [wandb_init.py:init():669] starting backend
2025-04-08 15:45:55,012 INFO    MainThread:8128 [wandb_init.py:init():673] sending inform_init request
2025-04-08 15:45:55,014 INFO    MainThread:8128 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2025-04-08 15:45:55,016 INFO    MainThread:8128 [wandb_init.py:init():686] backend started and connected
2025-04-08 15:45:55,022 INFO    MainThread:8128 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2025-04-08 15:45:55,022 INFO    MainThread:8128 [wandb_run.py:_label_probe_notebook():1349] Unable to probe notebook: 'charmap' codec can't decode byte 0x81 in position 6988: character maps to <undefined>
2025-04-08 15:45:55,022 INFO    MainThread:8128 [wandb_init.py:init():781] updated telemetry
2025-04-08 15:45:55,050 INFO    MainThread:8128 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2025-04-08 15:45:55,548 INFO    MainThread:8128 [wandb_init.py:init():867] starting run threads in backend
2025-04-08 15:45:55,891 INFO    MainThread:8128 [wandb_run.py:_console_start():2456] atexit reg
2025-04-08 15:45:55,891 INFO    MainThread:8128 [wandb_run.py:_redirect():2305] redirect: wrap_raw
2025-04-08 15:45:55,891 INFO    MainThread:8128 [wandb_run.py:_redirect():2370] Wrapping output streams.
2025-04-08 15:45:55,891 INFO    MainThread:8128 [wandb_run.py:_redirect():2395] Redirects installed.
2025-04-08 15:45:55,892 INFO    MainThread:8128 [wandb_init.py:init():911] run started, returning control to user process
2025-04-08 15:46:38,945 WARNING MsgRouterThr:8128 [router.py:message_loop():75] message_loop has been closed
